### PR TITLE
chore: add CLA verification

### PR DESCRIPTION
**Pull request by fedeci**
**What kind of change does this PR introduce? (Bug fix, feature, docs update, ...)**

**What is the current behavior? (You can also link to an open issue here)**

**What is the new behavior (if this is a feature change)?**

**Additional info:**

### CODE REVIEW

**Review by mschrage — APPROVED** [review]

**Comment by sasial-dev** [comment]
A CLA will work for all new PR's, but what about all the old code by the 300+ contributors who _haven't_ signed the CLA?

File: cla/cla.md [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+
+Thank you for your interest in software projects stewarded by Hercules Labs Inc. dba Fig (“Fig”). In order to clarify the intellectual property license granted with Contributions from any person or entity, Fig must have a Contributor License Agreement (CLA) on file that has been agreed to by each Contributor, indicating agreement to the license terms below. This license is for your protection as a Contributor as well as the protection of Fig and its users; it does not change your rights to use your own Contributions for any other purpose. This Agreement allows an individual to contribute to Fig on that individual’s own behalf, or an entity (the “Corporation”) to submit Contributions to Fig, to authorize Contributions submitted by its designated employees to Fig, and to grant copyright and patent licenses thereto.
+
+You accept and agree to the following terms and conditions for Your present and future Contributions submitted to Fig. Except for the license granted herein to Fig and recipients of software distributed by Fig, You reserve all right, title, and interest in and to Your Contributions.
```
Comment (brendanfalk) [inline cla/cla.md:7]:
```suggestion
You accept and agree to the following terms and conditions for Your Contributions submitted to Fig. Except for the license granted herein to Fig and recipients of software distributed by Fig, You reserve all right, title, and interest in and to Your Contributions.
```

**Review by brendanfalk — CHANGES_REQUESTED** [review]

**Review by brendanfalk — APPROVED** [review]

**Comment by brendanfalk** [comment]
> A CLA will work for all new PR's, but what about all the old code by the 300+ contributors who _haven't_ signed the CLA?

My understanding is the CLA applies to all contributions. In saying that, the CLA doesn't mean Fig "owns" your code. You still own your contributions and do what you want with them. The CLA just allows us to use these contributions for things in the future (like if we decide to relicense, or if we decide to use some of the withfig/autocomplete code in our own private codebase).

We spoke to a lot of very well known devtools founders recently and they all **highly** suggested we use a CLA. It's a quick easy process, very contributor friendly, and not doing it can only come back to bite us in the future.

File: .github/workflows/verify-cla.yml [hunk 2]
```diff
@@ -0,0 +1,25 @@
+name: Verify CLA
+on:
+  issue_comment:
+    types: [created]
```
[the remark this answers is not in the document]
Comment (fedeci) [reply .github/workflows/verify-cla.yml:4]: This is because PRs comments are issues. They get filtered later in the action.

**Pull request merged**